.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: ''
```

Comment:
Is there any value in ensuring that these issues all get a consistent label?

Reply:
Were you thinking of adding a `release` label? Or something else?

Reply:
That seems reasonable. Mostly suggesting so that people are aware that this is a feature of the issue template that exists. If we want it to be something in the future, we can modify it.

Reply:
That seems like a good idea so we can filter to find all issues related to a release